src/numeric/processes.jl

```diff
@@ -55,22 +83,31 @@ function discretize(mvn::MvNormal)
     DiscretizedIIDProcess(x, w)
 end
 
-
-# discrete markov processes
-
-type DiscreteMarkovProcess <: DiscreteProcess
-    transitions::Matrix{Float64}
-    values::Matrix{Float64}
+function rand(mvn::MvNormal, args...)
```

Comment:
Ok, that makes sense. I actually ran into this problem in QuantEcon and used some silly hack to get around it (see [here](https://github.com/QuantEcon/QuantEcon.jl/blob/cde1f395243151ce54f3f1ba6b7d079b751878d0/src/lss.jl#L26-L38)).

Given the updated comment about wrapping the distributions object, what do you think is the best way forward?

As of right now this method won't work as it is because cholesky doesn't accept a matrix of zeros, so whatever route we choose we still have to be clever in getting around that issue.